src/renderer/css/global.css

```diff
@@ -62,3 +62,17 @@ html.fa-events-icons-loading .fa-fw::before {
     -webkit-transition: color .4s ease;
     transition: color .4s ease;
 }
+
+.InstanceLogs-out {
+    font-family: monospace;
+    overflow-wrap: break-word;
+    border: rgba(100, 100, 100, 0.2) solid 1px;
+    padding: 3px;
+    margin: -1px;
+}
+
+.InstanceLogs-error {
+    color: red;
+    background-color: rgba(250, 116, 116, 0.623);
+    border: rgba(250, 116, 116, 0.774)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/manual/lukechu10/Minecraft-Box-Launcher/pullRequest?prid=5444487)